README.md

```diff
@@ -92,71 +194,35 @@ create_sfp(title = "titel van het protocol",
 ```
 
 
+## Rmarkdown
 
-## How to contribute?
-
-
-### Suggesting changes to existing protocols
-
-#### Using online editing functionalities
-
-To do: write this section
-
-
-#### Using RStudio and git version control
-
-Installing the repository on your local drive:
-
-* go to the repository https://github.com/inbo/protocols/ and press clone button
-* copy the URL to the clipboard
-* start RStudio and select `File -> New project -> Version Control -> Git` -> paste the URL
-* `protocols` will be automatically suggested as project directory name (keep it that way)
-* In the field `Create project as subdirectory of` select a folder on your local disc (do *not* use a folder that google drive file stream synchronizes). For instance `C:/R/GitRepositories`.
-* Click OK
-
-You will now have a local clone of the remote repository as an RStudio project. The file `protocols.Rproj` will be automatically created as well as a `.git` directory (do not make changes in this directory - it is where version control takes place). 
-The same directories and files which can be seen on the remote (https://github.com/inbo/protocols) will be copied to your local drive. 
-Whenever you want to work in the project, you need to double-click `protocols.Rproj` file to start the RStudio project. 
+Some useful resources for self-learning of **Rmarkdown**:
```

Comment:
Other suggestions: https://bookdown.org/yihui/rmarkdown/ and https://bookdown.org/yihui/bookdown/ . Rather as a reference.

Reply:
See 078cb8a